src/pages/rooms.tsx

```diff
@@ -0,0 +1,71 @@
+import { async } from "@firebase/util";
+import { NextPage } from "next";
+import React, { useEffect, useState } from "react";
+import { useStores } from "@/stores/context";
+import { type } from "os";
+import { observer } from "mobx-react-lite";
+import { RoomOverview } from "@/models/room/RoomOverview";
+const makeRoom: NextPage = observer(() => {
```

Comment:
페이지 혹은 컴포넌트는 대문자로 시작하도록 작성해와서 통일해야겠습니다.
```suggestion
const CreateRoom: NextPage = observer(() => {
```

Reply:
만들 뿐만 아니라 목록을 조회하기도 해서 ManageRooms로 변경하였습니다 !